scripts/bundle-copy.js

```diff
@@ -0,0 +1,37 @@
+// NOTE: this script is only necessary as long as we need to support
+// proxy exports for backwards compatibility.
```

Comment:
Unless we plan on including the png / svg images in `materials` in the final bundle, we will need to keep this, right?

Reply:
Yeah, but then I would it back to the rollup config once we have settled on our final structure.